mergify_engine/actions/queue.py

```diff
@@ -88,6 +87,17 @@ def get_config_schema(
             voluptuous.Required("require_branch_protection", default=True): bool,
         }
 
+        validator_commit_message = {
+            voluptuous.Required("commit_message", default="default"): voluptuous.Any(
+                "default", "title+body"
+            ),
+        }
+
+        schema = validator_default.copy()
```

Comment:
Unlike the merge action, validator_default is not a class variable here, it's local to the method, so why do you copy it?